### DISCUSSION TITLE
Disable Vanilla Door Interaction

### DISCUSSION THREAD

**Discussion by OverlordZorn** [Vdauphin/HeartsAndMinds#1400] [Question and answer]
Hi - 

At some point i read somewhere that you can disable vanilla actions on buildings for maps like Lythium where doors are bugged and cannot be locked via script.

Since ace allows people to open it anyway with, for example, CTRL + Spacebar, i came to the conclusion that its kind a pointless and i want to revert it back to the normal state, but i cannot find it anymore nore the resource where i found the original info.

**Comment by Vdauphin** [comment]
Hello,
Don't really understand what is the problem here
Cheers
